spec/features/submissions_spec.rb

```diff
@@ -197,6 +197,11 @@
           expect(page).to have_css(".has-error")
         end
 
+        scenario "sortowanie zgłoszeń względem daty nadesłania" do
+          visit "/submissions"
+          expect(page).to have_content(/01-01-2015.*01-01-2016/)
```

Comment:
Skoro sortowanie jest "desc" to pierwsze powinno być zgłoszenie o późniejszej dacie. Ponadto należy dwukrotnie kliknąć "Data nadesłania" i sprawdzić czy kolejność się zmienia.